app/components/views/GovernancePage/Proposals/index.js

```diff
@@ -54,15 +54,16 @@ function Proposals() {
   const preVoteCount = useSelector(sel.newPreVoteProposalsCount);
   const politeiaEnabled = useSelector(sel.politeiaEnabled);
   const location = useSelector(sel.location);
+  const dispatch = useDispatch();
 
   // TODO move reducers which only control local states from reducer/governance.js
   // to here.
   const [ tab, setTab ] = useReducer(() => getProposalsTab(location));
 
-  const dispatch = useDispatch();
   useEffect(() => {
     dispatch(setLastPoliteiaAccessTime());
-  }, []);
+  }, [ dispatch ]);
```

Comment:
Why you add this dispatch here? I left it empty to be used as a "componentDidMount"

Reply:
It's safer and obeys the `exhaustive-deps` rule. You can read more about it [here](https://github.com/facebook/react/issues/14920). The dispatch function is created outside the component context so it shouldn't change. It should work exactly the same.

Reply:
@tiagoalvesdulce thanks for reading my mind 👍

Reply:
😂😂😂